src/v2/spec.yaml

```diff
@@ -682,11 +684,141 @@ components:
             - cast
           properties:
             cast:
-              $ref: "#/components/schemas/CastWithInteractionsAndConversations"
+              $ref: '#/components/schemas/CastWithInteractionsAndConversations'
             chronological_parent_casts:
               type: array
               items:
-                $ref: "#/components/schemas/CastWithInteractions"
+                $ref: '#/components/schemas/CastWithInteractions'
+      example:
```

Comment:
Why do we need an example here?

Reply:
cast with interactionsconversation was breaking because its recursive, this is a workaround I found